packages/headless/src/features/sort-criteria/criteria.ts

```diff
@@ -1,22 +1,58 @@
 import {isArray} from '@coveo/bueno';
 
+/**
+ * The available sort order
```

Comment:
I believe it takes dots at the end of doc descriptions, here and in other descriptions